iosx/src/flowmgr/AVSFlowManager.m

```diff
@@ -1044,7 +1044,7 @@ - (int)setAudioEffect:(AVSAudioEffectType) effect
     }
     
     //    FLOWMGR_MARSHAL_RET(fmw.tid, ret, flowmgr_set_audio_effect, self.flowManager, effect_type);
-    
+#endif
```

Comment:
Since these are not used, maybe we can just remove the whole body of the function and return 0 ?